include/tvm/runtime/data_type.h

```diff
@@ -81,6 +82,10 @@ class DataType {
   bool is_float() const { return code() == DataType::kFloat; }
   /*! \return whether type is a float16 type. */
   bool is_float16() const { return is_float() && bits() == 16; }
+  /*! \return whether type is a bfloat type. */
+  bool is_bfloat() const { return code() == DataType::kBFloat; }
```

Comment:
given that only bfloat16 is defined, is_bf16 is a good enough function

Reply:
ok, changed